samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Shell.xaml.cs

```diff
@@ -277,4 +280,37 @@ private void NavigationViewControl_DisplayModeChanged(MUXC.NavigationView sender
 			}
 		}
 	}
+
+	public class MyLoadableSource : ILoadable
+	{
+
+		public event EventHandler? IsExecutingChanged;
```

Comment:
```suggestion
		public event EventHandler IsExecutingChanged;
```
so we dont keep getting this github actions check on subsequent prs